packages/alfa-rules/src/sia-r19/rule.ts

```diff
@@ -82,10 +82,10 @@ function isValid(attribute: aria.Attribute): boolean {
       return true;
 
     case "integer":
-      return /^\d+$/.test(value);
+      return /^-?\d+$/.test(value);
```

Comment:
```suggestion
      return /^[+-]?\d+$/.test(value);
```
Let's accept `+1` also, just in case.